tools/erc-repository-indexer/erc-contract-indexer/example.env

```diff
@@ -0,0 +1,3 @@
+HEDERA_NETWORK=
+MIRROR_NODE_URL=
```

Comment:
Do we need both `HEDERA_NETWORK` and `MIRROR_NODE_URL`? 

If we have a Hedera network, we might be able to create an SDK client via `Client.for<HEDERA_NETWORK>()`. Once we get the client, we will be able to get the mirror node URL by the already initialized client IMO.

Reply:
> Do we need both HEDERA_NETWORK and MIRROR_NODE_URL?

Yes, that's a great question! Currently, the script relies solely on the `MIRROR_NODE_URL` to retrieve contracts, so `HEDERA_NETWORK` could potentially be removed.

> If we have a Hedera network, we might be able to create an SDK client using `Client.for<HEDERA_NETWORK>()`. Once we obtain the client, we can retrieve the mirror node URL through the already initialized client, in my opinion.

That's a valid point! However, I don't see the necessity of using the SDK in this context. We aim to leverage the MN REST server and avoid making calls to the SDK, as those would ultimately route the requests to the mirror node anyway.